backend/src/modules/services/movies.js

```diff
@@ -9,37 +10,61 @@ const getMoviesByCategory = (category, sortBy, page) => {
   const pageParam = page ? `&page=${page}` : '';
   const requestPath = `${apiVersion}/movie/${category}?api_key=${apiKey}&language=en-US${sortParam}${pageParam}`;
 
-  return AxiosTheMovieDB.get(requestPath).then(({ data }) => data);
+  return AxiosTheMovieDB.get(requestPath)
+    .then(({ data }) => data)
+    .catch(({ response: { status, statusText } }) => {
+      generateError(status, statusText);
```

Comment:
What do you think if here we just pass the response to generateError and on the function definition we destructure what we need like:
```suggestion
      const generateError = ({ status, statusText }) => {...}
```

Reply:
Totally right! thanks!